examples/cluster/plot_digits_linkage.py

```diff
@@ -77,9 +70,9 @@ def plot_clustering(X_red, labels, title=None):
 
 for linkage in ("ward", "average", "complete", "single"):
     clustering = AgglomerativeClustering(linkage=linkage, n_clusters=10)
-    t0 = time()
+    t0 = perf_counter()
     clustering.fit(X_red)
-    print("%s :\t%.2fs" % (linkage, time() - t0))
+    print("%s :\t%.2fs" % (linkage, perf_counter() - t0))
```

Comment:
However, here we can use an f-string and the same in the title in `plot_clustering` below.